app/code/Magento/Inventory/Test/Integration/Indexer/SourceItemIndexerTest.php

```diff
@@ -97,4 +118,24 @@ public function testReindexAll()
         self::assertEquals(5, $this->getProductQuantityInStock->execute('SKU-2', 20));
         self::assertEquals(5, $this->getProductQuantityInStock->execute('SKU-2', 30));
     }
+
+    /**
+     * @param string $sku
+     * @param int $sourceId
+     *
+     * @return int
+     */
+    private function getInventorySourceItemId(string $sku, int $sourceId): int
```

Comment:
Looks like we can extract this code in separate object (in test namespace) and reuse this object in the other cases
After that we can remove `$connection->query('ALTER TABLE ' . $resourceConnection->getTableName('inventory_source_item') . ' AUTO_INCREMENT 1;');` from fixtures